gap/Algebroids.gi

```diff
@@ -919,6 +987,32 @@ InstallGlobalFunction( ADD_FUNCTIONS_FOR_RANDOM_METHODS_OF_ALGEBROID,
     
 end );
 
+BindGlobal( "AdditiveClosureOfAlgebroidPrecompiled", function ( A )
+  local Rq, is_finite_dimensional, is_quotient_of_path_algebra, over_Z;
+    
+    Rq := UnderlyingQuiverAlgebra( A );
+    
+    is_finite_dimensional := IsFiniteDimensional( Rq );
+    
+    is_quotient_of_path_algebra := IsQuotientOfPathAlgebra( Rq );
```

Comment:
Why only `is_quotient_of_path_algebra`? It is enough to be finite dimensional.

Reply:
I do not really understand your question. The code in Algebroids is different for quotients and non-quotients (because the former have to take `Representative`s at some places), so the two cases produce different precompiled code -> I have to differentiate the two cases. If you would like to have a precompiled version of a non-quotient version: this is not a problem, see point 2. in the PR description.

Reply:
Sorry, I was missing the context.